azure-ai-speech-toolkit/src/handlers.ts

```diff
@@ -438,7 +461,7 @@ export async function downloadSampleApp(...args: unknown[]) {
       console.log("Created parent folder: " + selectedFolder);
     }
 
-    console.log("projectPath: " + projectPath);
+    telemetryReporter.sendTelemetryEvent("test-2-1752");
```

Comment:
What's this?

Reply:
It is a test telemetry, will remove in my next PR